code/modules/mining/equipment/mining_tools.dm

```diff
@@ -16,8 +15,12 @@
 	tool_behaviour = TOOL_MINING
 	toolspeed = 1
 	usesound = list('sound/effects/picaxe1.ogg', 'sound/effects/picaxe2.ogg', 'sound/effects/picaxe3.ogg')
-	attack_verb_continuous = list("hits", "pierces", "slices", "attacks")
-	attack_verb_simple = list("hit", "pierce", "slice", "attack")
+	attack_verb_simple = list("hit", "pierce", "mine", "attack")
+	attack_verb_continuous = list("hits", "pierces", "mines", "attacks")
+	force = 15
+	armour_penetration = 25
+	wound_bonus = 30
```

Comment:
this is high enough to guarantee at least a basic puncture wound on the first hit, and at least a second degree puntue wound on the second hit.

Reply:
I wanted the pickaxes to be able to work their way through the entire stack of puncture wounds and bone wounds, since they are pointy tools meant for breaking rock. This means that they need a significant bonus to wounding power. When I did the math this came out to at least 21 wound bonus for a compound fracture assuming that there was already a critical puncture wound (required for a `SHARP_PIERCING` object to deal bone wounds) and a _prefect wounding roll_. I upped it to 30 when I came to the conclusion that a 1/(20^1.4) chance was too low.